edward/stats/distributions.py

```diff
@@ -9,114 +9,83 @@ class Distribution:
     """Template for all distributions."""
     def rvs(self, size=1):
         """
-        Parameters
-        ----------
-        size : int, list, or tuple, optional
-            Number of samples, in a particular shape if specified in a
-            list or tuple with more than one element.
-
-        params : float or np.array
-
         Returns
         -------
         np.ndarray
-            np.array of dimension (size x shape), where shape is the
-            shape of its parameter argument. For multivariate
-            distributions, shape may correspond to only one of the
-            parameter arguments, e.g., alpha in Dirichlet, p in
-            Multinomial, mean in Multivariate_Normal.
+            size-dimensional vector; scalar if size=1
 
         Notes
         -----
         This is written in NumPy/SciPy, as TensorFlow does not support
-        many distributions for random number generation. It follows
-        SciPy's naming and argument conventions.
-
-        The equivalent method in SciPy is not guaranteed to be
-        supported with a batch of parameter inputs, e.g., a vector of
-        location parameters in a normal distribution, or a matrix of
-        concentration parameters in a Dirichlet. This is.
-
-        This does not follow SciPy's behavior, e.g., the number (or
-        shape) of the draws will always be denoted by its outer
-        dimension(s).
-
-        params as a 2d or higher tensor is not guaranteed to be
-        supported (for either univariate or multivariate
-        distribution).
-
-        size as a list or tuple of more than one element is not
-        guaranteed to be supported.
-
-        For most distributions, the parameters must be of the same
-        shape and type, e.g., n and p in Binomial must be np.arrays()
-        of same shape or both floats. For some, they may differ by one
-        dimension, e.g., n and p in Multinomial can be float and
-        np.array(), or both np.arrays, and n always has one less
-        dimension.
+        many distributions for random number generation.
         """
         raise NotImplementedError()
 
     def logpmf(self, x):
         """
         Parameters
         ---------
-        x : float, np.array, or tf.Tensor
-            If univariate distribution, can be a scalar or tensor.
-            If multivariate distribution, can be a tensor; the outer
-            dimension carries the multivariate dimension.
+        x : np.array or tf.Tensor
+            If univariate distribution, can be a scalar, vector, or matrix.
+            If multivariate distribution, can be a vector or matrix.
 
-        params : float, np.array, or tf.Tensor
+        params : np.array or tf.Tensor
```

Comment:
This should keep `float, np.array or tf.Tensor`. parameters of type `float` is supported